addons/mass_mailing/static/src/js/mass_mailing_widget.js

```diff
@@ -299,6 +299,7 @@ var MassMailingFieldHtml = FieldHtml.extend({
             }
         }
         this.wysiwyg.trigger('reload_snippet_dropzones');
+        core.bus.trigger('iframe_updated', this.wysiwyg.$iframe);
```

Comment:
I'm wondering, since we are in a widget and want to notify our form view, wouldn't a "good old" trigger_up do the trick, instead of using the bus?